src/Confluent.Kafka/Producer.cs

```diff
@@ -322,23 +418,11 @@ public Producer(IEnumerable<KeyValuePair<string, object>> config, bool manualPol
         ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md).
         /// </param>
         public Producer(IEnumerable<KeyValuePair<string, object>> config)
+#pragma warning disable CS0618
```

Comment:
disable obsoleted method warning. todo: refactor, when that constructor is removed.